safenode/src/network/event.rs

```diff
@@ -104,9 +104,9 @@ impl SwarmDriver {
                         let new_peers: HashSet<PeerId> = closest_peers.peers.into_iter().collect();
```

Comment:
Sorry, couldn't actually tag the line I wanted. Anways, in L116, we are just logging a trace msg if the `pending_get_closest_peers.remove()` returns `None`, maybe we can use `ReceivedResponseDropped` here as well?

Reply:
makes sense